### PR TITLE
#1475: Fixed mapping expansions for profile-informed authoring not re…

### PR DESCRIPTION
**Pull request by brynrhodes**
…specting signature level

### CODE REVIEW

**Review by JPercival — APPROVED** [review]

File: Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java [hunk 1]
```diff
@@ -2743,6 +2743,22 @@ public Expression applyTargetMap(Expression source, String targetMap) {
             String functionArgument = targetMap.substring(invocationStart + 1, targetMap.lastIndexOf(')'));
             Expression argumentSource =
                     functionArgument.equals("%value") ? source : applyTargetMap(source, functionArgument);
+
+            // NOTE: This is needed to work around the mapping for ToInterval
```
Comment (lukedegruchy) [inline Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java:2747]: General feedback:  Please fix the title and description of this PR so at a minimum it's not a truncated version of the commit message.

File: Src/java/cql-to-elm/src/test/java/org/cqframework/cql/cql2elm/TranslationTests.java [hunk 2]
```diff
@@ -467,4 +467,38 @@ void abstractClassNotRetrievable() throws IOException {
                 errors.stream().map(Throwable::getMessage).collect(Collectors.toList());
         assertThat(errorMessages, contains("Specified data type DomainResource does not support retrieval."));
     }
+
+    @Test
+    void mappingExpansionsRespectSignatureLevel() throws IOException {
```
Comment (lukedegruchy) [inline Src/java/cql-to-elm/src/test/java/org/cqframework/cql/cql2elm/TranslationTests.java:472]: Please add a test for Signature.None to contrast with the test below.  Also, it might be good to test other non-Overloads, non-None Signature types as well.

**Review by lukedegruchy — APPROVED** [review]
